wgsl/index.bs

```diff
@@ -11546,14 +11593,21 @@ The [=texel format=] names are:
     | `'bgra8unorm'`
 </div>
 
-The [=extension=] names are:
+The [=enable-extension=] names are:
 
 <div class='syntax' noexport='true'>
-  <dfn for=syntax>extension_name</dfn> :
+  <dfn for=syntax>enable_extension_name</dfn> :
 
     | `'f16'`
 </div>
 
+There are no [=software extension=] names, but they have the same form as an [=identifier=]:
+<div class='syntax' noexport='true'>
+  <dfn for=syntax>software_extension_name</dfn> :
+
+    | [=syntax/ident_pattern_token=]
```

Comment:
I see, this is great. Can't other extension types also adopt this and avoid changing syntax for each new hardware feature?

Reply:
Yes, I think enable-extensions (as this PR calls them) can use ident_pattern_token.
Then use the table in the spec as the source of truth for what extensions are allowed.

Reply:
I meant same for enable extensions, I apologize for my unclear wording.